gt2_test_runner/__init__.py

```diff
@@ -546,7 +553,27 @@ def coverage_report(self, save_data=False, html_dir=None, to_stream=True):
 
             print('\nHTML coverage data is saved as file://{}/index.html'.format(html_dir),
                   file=self.stream)
+    
+    def setup_buffer(self, test):
+        sys.stdout = self.test_data[test_fqn]['standard out'] = io.StringIO()
+        sys.stderror = self.test_data[test_fqn]['standard error'] = io.StringIO()
```

Comment:
This should be `stderr`, not `stderror`. You won’t get an error, but it won’t do what you try to do…